gammapy/image/maps.py

```diff
@@ -489,6 +531,44 @@ def to_sherpa_data2d(self, dstype='Data2D'):
         else:
             raise ValueError('Invalid sherpa data type.')
 
+    def upsample(self, factor, order=3, shape=None):
+        """
+        Up sky map by a power of two.
+
+        The sky map is up sampled using `scipy.ndimage.zoom`. Only
+        up sampling factors, that are a power of two are allowed. The sky map is
+        cropped to a given size.
+
+        Parameters
+        ----------
+        factor : int
+            up sampling factor, must be power of two.
+        order : np.ufunc (np.nansum), optional
```

Comment:
fix parameter description: order is an int that's passed to
http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.ndimage.interpolation.zoom.html